### DISCUSSION TITLE
output package from Qgis to Qfield giving errors

### DISCUSSION THREAD

**Discussion by Belgravekm** [opengisch/QField#3666] [Q&A]
Hello,
I have installed Qgis 3.28 installed and when I try to use qsync i keep getting the same set of errors.
[Qfield packaging errors.docx]  here is a screen shot of the errors
(https://github.com/opengisch/QField/files/10051295/Qfield.packaging.errors.docx)
Thankx

**Comment by nirvn** [comment]
@Belgravekm , the error message indicates you are running a _very_ old version of qfieldsync (roughly 3.2.2). This is not supported anymore, please update qfieldsync to the latest official version and try again.

**Comment by Belgravekm** [comment]
Thank you will do.

**Comment by Belgravekm** [comment]
Hello,
 Thankx again for the info it worked, but I decided to update my qgis along with the qfield sync, and now I am getting a fresh error. sigh please help
[QField error.docx](https://github.com/opengisch/QField/files/10087927/QField.error.docx)
